deployment/charts/templates/deploy.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.app.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-service-monitor
+  labels:
+    release: {{ .Values.app.prometheus.helm_release_name }}
+spec:
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
```

Comment:
Ideally we want to use more labels under `selector` to make sure we target the right thing. When doing multiple deployments this is going to target all of them, unless `.Values.nameOverride` si specified and it's going to result in unexpected behaviours

Reply:
added `release: {{ .Release.Name }}`